README.md

```diff
@@ -1 +1,48 @@
-# forecasting-live-demo-backend
\ No newline at end of file
+# Built on Vertex AI: Backend
+
+This is the backend code for "Built on Vertex AI".
+
+It uses the following packages:
+
+- [FastAPI](https://fastapi.tiangolo.com/)
+- [Google Vertex AI SDK for Python](https://cloud.google.com/vertex-ai/docs/start/use-vertex-ai-python-sdk#install-vertex-ai-python-sdk)
+- [Google BigQuery SDK for Python](https://github.com/googleapis/python-bigquery)
+
+## Available Commands
+
+In the project directory, you can run:
+
+### Deployment
+
+Here are a few options on how to deploy the backend to your own Google Cloud project.
+
+#### 1. Build and deploy to Google Cloud Run in one step
+
+```
+gcloud beta run deploy built-on-vertex-ai-backend --source . --region=us-central1 --platform=managed --cpu=2 --memory=8G --timeout=3600 --allow-unauthenticated --session-affinity --min-instances=1
```

Comment:
In what directory should the user run this command? Asking because of that dot (.) in the command.

Reply:
In the same folder as the README. I'll add a note